core/controllers/base_test.py

```diff
@@ -1423,3 +1469,397 @@ def test_that_correct_derived_class_does_not_raise_exception(self):
         with self.swap(self, 'testapp', self.mock_testapp):
             self.post_json(
                 '/correctmock', payload, expected_status_int=200)
+
+
+class SchemaValidationIntegrationTests(test_utils.GenericTestBase):
+    """Tests all the functionality of SVS(Schema-Validation-System)
+    architecture.
+    """
+    handler_class_names_with_no_schema = (
+        payload_validator.HANDLER_CLASS_NAMES_WITH_NO_SCHEMA)
+    wiki_page_link = (
+        'https://github.com/oppia/oppia/wiki/Validation-of-handler-args')
+
+    def _get_list_of_routes_which_need_schemas(self):
+        """This method iterates over all the routes and returns those routes
+        which need schema.
+
+        Returns:
+            list_of_routes_which_needs_schema: list(RedirectRoute). A list of
+            RedirectRoute objects.
+        """
+        list_of_routes_which_needs_schema = []
```

Comment:
needs --> need (since "routes" is plural)
schema --> schemas

Reply:
Done